pkg/utils/config.go

```diff
@@ -0,0 +1,100 @@
+package utils
+
+import (
+	"github.com/Unknwon/goconfig"
```

Comment:
This library is not maintained, you can consider the other library, such as [ini](https://github.com/go-ini/ini)

Reply:
@wisererik good suggestion, the main authors of both libraries are Unknown, there wold be many similarity between the two libraries. and it seems that [INI](https://github.com/go-ini/ini) community is more active and more popular.